etna/collections/models.py

```diff
@@ -400,16 +413,29 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
 
     @cached_property
     def related_articles(self):
-        from etna.articles.models import ArticlePage
+        """Return a list of related pages for rendering in the related articles section
+        of the page. To add another page type, import it and add it to the related_page_types list.
+        """
 
-        return (
-            ArticlePage.objects.exclude(pk=self.featured_article)
-            .live()
-            .public()
-            .filter(pk__in=self.related_page_pks)
-            .order_by("-first_published_at")
-            .select_related("teaser_image")
-        )
+        from etna.articles.models import ArticlePage, FocusedArticlePage
+
+        related_page_types = [ArticlePage, FocusedArticlePage]
+        page_list = []
+
+        for page_type in related_page_types:
+            pages = (
+                page_type.objects.exclude(pk=self.featured_article_id)
+                .filter(pk__in=self.related_page_pks)
+                .live()
+                .public()
+                .select_related("teaser_image")
+            )
```

Comment:
I would shorten this whole thing to:
```suggestion
        page_list = []
        for page_type in [ArticlePage, FocusedArticlePage]:
            page_list.extend(
                page_type.objects.exclude(pk=self.featured_article_id)
                .filter(pk__in=self.related_page_pks)
                .live()
                .public()
                .select_related("teaser_image")
                .prefetch_related("teaser_image__renditions")
            )
            return sorted(page_list, key=lambda x: x.first_published_at, reverse=True)
```

Reply:
Hi Andy, thanks for all these suggestions :) Do I still need the `prefetch_related` as well as `select_related`?